globus_automate_client/action_client.py

```diff
@@ -84,6 +84,8 @@ def run(
             "monitor_by": monitor_by,
             "manage_by": manage_by,
         }
+        # Remove None / empty list items from the temp_body
+        body = {k: v for k, v in body.items() if v}
```

Comment:
But, what if `v` is actually the value `False`? Or what if it is the empty-string?